xml/System.Text.Json/JsonSerializerOptions.xml

```diff
@@ -782,9 +782,14 @@ For more information, see [How to ignore properties with System.Text.Json](/dotn
         <Parameter Name="populateMissingResolver" Type="System.Boolean" Index="0" FrameworkAlternate="net-8.0" />
       </Parameters>
       <Docs>
-        <param name="populateMissingResolver">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="populateMissingResolver">Populates unconfigured <see cref="P:System.Text.Json.JsonSerializerOptions.TypeInfoResolver" /> properties with the reflection-based default.</param>
+        <summary>Marks the current instance as read-only preventing any further user modification.</summary>
+        <remarks>When <paramref name="populateMissingResolver" /> is set to <see langword="true" />, configures the instance following the semantics of the <see cref="T:System.Text.Json.JsonSerializer" /> methods accepting <see cref="T:System.Text.Json.JsonSerializerOptions" /> parameters.
+ 
+ This method is idempotent.</remarks>
+        <exception cref="T:System.InvalidOperationException">The instance does not specify a <see cref="P:System.Text.Json.JsonSerializerOptions.TypeInfoResolver" /> setting. Thrown if <paramref name="populateMissingResolver" /> is <see langword="false" />.
+             -OR-
+             The <see cref="P:System.Text.Json.JsonSerializer.IsReflectionEnabledByDefault" /> feature switch has been turned off.</exception>
```

Comment:
It's actually 3 cases, no?:

```suggestion
        <exception cref="T:System.InvalidOperationException">
          <para>The instance does not specify a <see cref="P:System.Text.Json.JsonSerializerOptions.TypeInfoResolver" /> setting.</para>
          <para>-or-</para>
          <para><paramref name="populateMissingResolver" /> is <see langword="false" />.</para>
          <para>-or-</para>
          <para>The <see cref="P:System.Text.Json.JsonSerializer.IsReflectionEnabledByDefault" /> feature switch has been turned off.</para>
        </exception>
```

Reply:
No. The first case only happens if `populateMissingResolver` is `false`. If it's true it will populate the `TypeInfoResolver` itself.

Reply:
Ok then let me adjust the suggestion:

```suggestion
        <exception cref="T:System.InvalidOperationException">
          <para>The instance does not specify a <see cref="P:System.Text.Json.JsonSerializerOptions.TypeInfoResolver" /> setting.</para>. Thrown when <paramref name="populateMissingResolver" /> is <see langword="false" />.</para>
          <para>-or-</para>
          <para>The <see cref="P:System.Text.Json.JsonSerializer.IsReflectionEnabledByDefault" /> feature switch has been turned off.</para>
        </exception>
```